0_overview.md

```diff
@@ -66,3 +66,63 @@ Cowboy is an HTTP server written in Erlang by Loïc Hoguin of [99s](http://ninen
 Cowboy has fantastic documentation. The [Guides](http://ninenines.eu/docs/en/cowboy/HEAD/guide/) are especially helpful. Learning more about Cowboy will surely help you to understand Phoenix more fully.
 
 Cowboy has its own section of links in the [Resources Guide](http://www.phoenixframework.org/docs/resources).
+
+
+### Dependencies
```

Comment:
What if we call this "System dependencies"? What I mean is, dependencies you need to install on your machine and that are not brought as part of Phoenix. We can start with the operating system, saying we support all of them. And then continue as is?